sonarqube/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM sonarqube
+
+COPY --chown=sonarqube:sonarqube docker-healthcheck /usr/local/bin/
```

Comment:
What's the purpose of the `--chown=` here?

Reply:
a mistake :) i'm removing it from the PR. nice catch